ts/components/CustomBadge.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * A component to render the circolar badge
+ * TODO: use the same component on all lists (messages, services, transaction)
```

Comment:
Every TODO must be associated with a pivotal story.

Reply:
Ok